recipe/meta.yaml

```diff
@@ -1,72 +1,100 @@
-{% set version = "2.4.12" %}
-{% set filename = "opencv2-%s.zip" % version %}
+{% set version = "3.1.0" %}
 
 package:
   name: opencv
   version: {{ version }}
 
 source:
-  fn: {{ filename }}
-  url: https://github.com/Itseez/opencv/archive/{{ version }}.zip
-  sha1: 2225768a48dbf99426bf77451a433ed432f54d8e
+  fn: opencv-{{ version }}.tar.gz
+  url: https://github.com/Itseez/opencv/archive/{{ version }}.tar.gz
```

Comment:
I guess this is now `https://github.com/opencv/opencv/archive/{{ version }}.tar.gz`